tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -498,7 +501,7 @@ def _test_convolution(
     add_shapes_to_graph_def=True,
 ):
     """One iteration of convolution with given shapes and attributes"""
-
+    # pylint: disable=dangerous-default-value
```

Comment:
please don't disable this one, it can lead to very insidious false test results. here and below.

Reply:
How can i pass ci without disable it. https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-12028/51/pipeline/
@areusch

Reply:
You have a mutable default argument here:
[def _test_convolutio](https://github.com/apache/tvm/blob/45b2d39d8fce7a11a596d03c6b914aa47fc2f952/tests/python/frontend/tensorflow/test_forward.py#L495)

If you want to know why this is bad, look for example here: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

To resolve this issue change
deconv_output_shape=[] to deconv_output_shape=None

and add 
deconv_output_shape = deconv_output_shape or []
add the start of the function.

Reply:
Thanks for your answer, learned a lot